scripts/support/pyocd.sh

```diff
@@ -9,6 +9,11 @@ BIN_NAME=${O}/${KERNEL_BIN_NAME}
 ELF_NAME=${O}/${KERNEL_ELF_NAME}
 GDB_PORT=${GDB_PORT:-3333}
 
+PYOCD_BOARD_ID_ARG=""
+if [ -n "${PYOCD_BOARD_ID}" ]; then
+	PYOCD_BOARD_ID_ARG="-b ${PYOCD_BOARD_ID}"
+fi
```

Comment:
It's typical to use Alternate Value substitution in the above case:
PYOCD_BOARD_ID_ARG=${PYOCD_BOARD_ID:+-b $PYOCD_BOARD_ID}